packages/create-redwood-app/template/api/src/lib/logger.ts

```diff
@@ -0,0 +1,88 @@
+import { createLogger, defaultLoggerOptions } from '@redwoodjs/api/logger'
+
+/**
+ * Creates a logger. Options define how to log. Destination defines where to log.
+ * If no destination, std out.
+ */
+export const logger = createLogger({
+  options: { ...defaultLoggerOptions },
+})
```

Comment:
Maybe we can just leave these as the default in the function itself?

Reply:
I thought about that, but I really like explicitly showing it here because you gets some nice help:

![image](https://user-images.githubusercontent.com/1051633/110323459-a30ff300-7fe2-11eb-8ea1-c954e268fb78.png)

and 

![image](https://user-images.githubusercontent.com/1051633/110323542-b9b64a00-7fe2-11eb-8686-f1339bdbfc95.png)


which I think help you know what the defaults are and the expected config out-of-the-box.

What do you think, @peterp ? Happy to move though.

Reply:
Ah some typos ... I fix!

Reply:
You can actually make that part of the type, then it'll be auto-completed and typed checked. I think the comment will also show up.

Reply:
Would you be OK with making the `defaultLoggerOptions` part of the function, and using types to document them to the user during design time?

If so, and if you need any guidance to make this happen let me know.

Reply:
@peterp 

> Would you be OK with making the defaultLoggerOptions part of the function,

I think so.

> if you need any guidance to make this happen let me know.

I'm not sure I understand what is needed to do that though.

```js
export const createLogger = ({
  options,
  destination,
  showConfig = isDevelopment,
}: RedwoodLoggerOptions): BaseLogger => {
```

Include the `defaultLoggerOptions` always? And then just override with `options` that are `LoggerOptions`?

Like

```js
export const createLogger = ({
  options,
  destination,
  showConfig = isDevelopment,
}: RedwoodLoggerOptions): BaseLogger => {

const loggerOptions = { ... defaultLoggerOptions, options}

```

and then use `loggerOptions` in the function?

That way you'd declare a logger like:

```js
export const logger = createLogger()
```

But if you do want to override:

```js
export const logger = createLogger({options: { level: 'warn' })
```

?